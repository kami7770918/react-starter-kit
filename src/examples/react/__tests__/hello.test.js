import * as React from 'react';
import renderer from 'react-test-renderer';
import {Hello} from '../hello';

test('renders correctly', () => {
  const tree = renderer.create(
    <Hello name="John" />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
