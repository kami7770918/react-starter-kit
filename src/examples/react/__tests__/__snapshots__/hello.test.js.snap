// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <img
    alt=""
    src=""
    style={
      Object {
        "width": "20px",
      }
    }
  />
  <div>
    Hello 
    John
  </div>
  <button
    onClick={[Function]}
  >
    toggle: 
    ON
  </button>
</div>
`;
